source/_patterns/molecules/interactive/accordion/accordion.twig

```diff
@@ -0,0 +1,24 @@
+{% set tag = tag|default('h2') %}
+{% set classes = [
+  'usa-accordion',
+  'usa-accordion--bordered'
+  ]|merge(classes|default([]))
+%}
+
+<div{{ attributes.addClass(classes) }}>
+  {% for item in accordion %}
+    {% set expanded = loop.first ? 'true' : 'false' %}
```

Comment:
Twig has some nice variables available when working in [for](https://twig.symfony.com/doc/1.x/tags/for.html) loops.  Letting the component template handle this is better UX.